src/main/java/christmas/config/AppConfig.java

```diff
@@ -0,0 +1,167 @@
+            menu.register(dessertCategory());
+            return menu;
+        }
+
+        private static List<Event> createEvents() {
+            List<Event> events = new ArrayList<>();
+            events.add(christmasDdayEvent());
+            events.add(specialEvent());
+            events.add(weekendEvent());
+            events.add(weekdayEvent());
+            events.add(giveawayEvent());
+            return events;
+        }
+
+        private static AppetizerCategory appetizerCategory() {
+            AppetizerCategory appetizerCategory = new AppetizerCategory();
+            appetizerCategory.register("양송이수프", 6000);
+            appetizerCategory.register("타파스", 5500);
+            appetizerCategory.register("시저샐러드", 8000);
+            return appetizerCategory;
+        }
+
+        private static DrinkCategory drinkCategory() {
+            DrinkCategory drinkCategory = new DrinkCategory();
+            drinkCategory.register("제로콜라", 3000);
+            drinkCategory.register("레드와인", 60_000);
+            drinkCategory.register("샴페인", 25_000);
+            return drinkCategory;
+        }
+
+        private static MainCategory mainCategory() {
+            MainCategory mainCategory = new MainCategory();
+            mainCategory.register("티본스테이크", 55_000);
+            mainCategory.register("바비큐립", 54_000);
+            mainCategory.register("해산물파스타", 35_000);
+            mainCategory.register("크리스마스파스타", 25_000);
+            return mainCategory;
+        }
+
+        private static DessertCategory dessertCategory() {
+            DessertCategory dessertCategory = new DessertCategory();
+            dessertCategory.register("초코케이크", 15_000);
+            dessertCategory.register("아이스크림", 5_000);
+            return dessertCategory;
+        }
+
+        private static Event weekendEvent() {
+            final int WEEKEND_EVENT_PRICE = 2023;
+
+            return new WeekendEvent(
+                    EVENT_START_DATE,
+                    EVENT_END_DATE,
+                    dessertCategory(),
+                    new FixDiscountPolicy(WEEKEND_EVENT_PRICE));
+        }
+
+        private static Event weekdayEvent() {
+            final int WEEKDAY_EVENT_PRICE = 2023;
+
+            return new WeekdayEvent(
+                    EVENT_START_DATE,
+                    EVENT_END_DATE,
+                    mainCategory(),
+                    new FixDiscountPolicy(WEEKDAY_EVENT_PRICE));
+        }
+
+        private static Event specialEvent() {
+            final int SPECIAL_EVENT_PRICE = 1000;
+            final List<Integer> STAR_DAYS = List.of(3, 10, 17, 24, 25, 31);
+
+            return new SpecialEvent(EVENT_START_DATE,
+                    EVENT_END_DATE,
+                    STAR_DAYS,
+                    new FixDiscountPolicy(SPECIAL_EVENT_PRICE));
+        }
+
+        private static Event giveawayEvent() {
+            List<Dish> dishes = new ArrayList<>();
+            dishes.add(Drink.of("샴페인", 25_000));
```

Comment:
```suggestion
            List<Dish> dishes = List.of(Drink.of("샴페인", 25_000));
```
`List.of`를 활용하면 불변 리스트를 더 쉽고 깔끔하게 만들 수 있어 좋은 것 같아요